src/components/AboutComponent.vue

```diff
@@ -34,9 +34,10 @@
          <li class="neon-grey-2 text-left text-0xl">2nd Place Team: <strong>$512</strong></li>
          <li class="neon-brown text-left text-xl">3rd Place Team: <strong>$256</strong></li>
        </ul>
-       <p class="text-xl text-white text-left">There are also subscriptions to services and swag in the mix! You must sign up by <strong>February 11, 2021</strong> and have your Shipping Address in your account in order to receive swag.</p>
-       <p class="text-xl text-white text-left">Note: Only teams consisting of all students may be eligible for prizes.</p> -->
-       <p class="text-2xl text-white text-left">Information about prizes will be available soon!</p>
+       <p class="text-xl text-white text-left">There are also subscriptions to services and swag in the mix! You must sign up by <strong>February 11, 2021</strong> and have your Shipping Address in your account in order to receive swag.</p> -->
+       <p class="text-2xl text-white text-left">Prizes will be given out to the top 5 teams. We will also be giving out swag to players joining us in person this year!</p>
+       <p class="text-2x1 text-white text-left">The nature of the prizes will be revealed soon! </p>
```

Comment:
the issue is that the class here is specified to be `text-2x1` (with a one (1) ) rather than the other `p` elements using a `text-2xl` (with a lower-cased letter L (l) ).

Reply:
```suggestion
       <p class="text-2xl text-white text-left">The nature of the prizes will be revealed soon! </p>
```

Reply:
beyond that this looks good to me

Reply:
Just updated it